src/AzureExtension/DevBox/Constants.cs

```diff
@@ -88,31 +88,64 @@ public static class Constants
 
     public const string DevBoxRepairOperation = "repair";
 
-    public const string DevBoxCreatingProvisioningState = "Creating";
+    public const string DevBoxDeletedState = "Deleted";
 
-    public const string DevBoxProvisioningState = "Provisioning";
+    public const string DevBoxOperationNotStartedState = "NotStarted";
 
-    public const string DevBoxProvisioningFailedState = "ProvisioningFailed";
+    public static class DevBoxProvisioningStates

```

Comment:
I'm surprised style cop isn't erroring out that classes should be in separate files

Reply:
I was pleasantly surprised as well. :)